src/components/NotesView/NotesView.tsx

```diff
@@ -0,0 +1,61 @@
+import { IconButton } from "@fluentui/react";
+import { Editor } from '@tinymce/tinymce-react';
+import React, { FunctionComponent, useState } from "react";
+import { Button, Col, Row, Spinner } from "react-bootstrap";
+import { INote } from "../../api/DomainObjects";
+import './NotesView.css';
+import { NoteView } from "./NoteView";
+
+export interface INotesViewProps {
+    notes: INote[],
+    submitNote: (text: string) => Promise<INote | undefined>,
+    className?: string
+}
+
+export const NotesView: FunctionComponent<INotesViewProps> = (props) => {
+
+    const [showNewNoteEditor, setShowNewNoteEditor] = useState<boolean>(false);
+    const [editorContent, setEditorContent] = useState<string>('');
+    const [submitting, setSubmitting] = useState<boolean>(false);
+
+    const submitNote = async () => {
+        setSubmitting(true);
+        await props.submitNote(editorContent);
+        setSubmitting(false);
+        setEditorContent('');
+        setShowNewNoteEditor(false);
+    }
+
+    return (
+        <div className={props.className}>
+            <Row className="ml-0 mr-2">
+                <h5>Notes</h5>
+                <span className="ml-auto mr-1 add-note-text">Add a Note</span>
+                <IconButton iconProps={{ iconName: "BoxAdditionSolid", className: "notes-button-icon" }} onClick={() => setShowNewNoteEditor(true)} />
+            </Row>
+            {showNewNoteEditor &&
+                <>
+                    <div className="mt-3 mb-3">
+                        <Editor
+                            init={{
+                                height: '20rem',
+                                menubar: false,
+                                plugins: [
+                                    'advlist autolink lists link charmap anchor',
+                                    'searchreplace visualblocks',
+                                    'insertdatetime paste wordcount'
```

Comment:
Likely need to turn this into a saved config item for reuse elsewhere so our RTE appears consistent